src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/Atom10ItemFormatter.cs

```diff
@@ -2,63 +2,53 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Globalization;
-using System.Text;
 using System.Xml;
 using System.Xml.Serialization;
 using System.Diagnostics.CodeAnalysis;
 using System.Xml.Schema;
-using System.Runtime.CompilerServices;
 
 namespace System.ServiceModel.Syndication
 {
     [XmlRoot(ElementName = Atom10Constants.EntryTag, Namespace = Atom10Constants.Atom10Namespace)]
     public class Atom10ItemFormatter : SyndicationItemFormatter, IXmlSerializable
     {
         private Atom10FeedFormatter _feedSerializer;
-        private Type _itemType;
-        private bool _preserveAttributeExtensions;
-        private bool _preserveElementExtensions;
+        private bool _preserveAttributeExtensions = true;
+        private bool _preserveElementExtensions = true;
 
-        public Atom10ItemFormatter()
-            : this(typeof(SyndicationItem))
+        public Atom10ItemFormatter() : this(typeof(SyndicationItem))
         {
         }
 
-        public Atom10ItemFormatter(Type itemTypeToCreate)
-            : base()
+        public Atom10ItemFormatter(Type itemTypeToCreate) : base()
         {
             if (itemTypeToCreate == null)
             {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull("itemTypeToCreate");
+                throw new ArgumentNullException(nameof(itemTypeToCreate));
             }
             if (!typeof(SyndicationItem).IsAssignableFrom(itemTypeToCreate))
             {
-                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgument("itemTypeToCreate",
-                    SR.Format(SR.InvalidObjectTypePassed, "itemTypeToCreate", "SyndicationItem"));
+                throw new ArgumentException(SR.Format(SR.InvalidObjectTypePassed, nameof(itemTypeToCreate), nameof(SyndicationItem)), nameof(itemTypeToCreate));
             }
+
             _feedSerializer = new Atom10FeedFormatter();
-            _feedSerializer.PreserveAttributeExtensions = _preserveAttributeExtensions = true;
-            _feedSerializer.PreserveElementExtensions = _preserveElementExtensions = true;
```

Comment:
After your change, you are no longer setting _feedSerializer.PreserveAttributeExtensions and _feedSerializer.PreserveElementExtensions to true. This is changing behavior.

Reply:
It isn't, as `true` is already the default:
```

        [Fact]
        public void Ctor_Default()
        {
            var formatter = new Atom10FeedFormatter();
            Assert.Null(formatter.Feed);
            Assert.True(formatter.PreserveAttributeExtensions);
            Assert.True(formatter.PreserveElementExtensions);
            Assert.Equal("Atom10", formatter.Version);
        }
```